packages/common/refresh-runtime.js

```diff
@@ -653,10 +653,7 @@ export function validateRefreshBoundaryAndEnqueueUpdate(
 
 function predicateOnExport(ignoredExports, moduleExports, predicate) {
   for (const key in moduleExports) {
-    if (key === '__esModule') continue
```

Comment:
Since `moduleExports` is a module namespace, the `__esModule` marker won't exist. If you have `export const __esModule = something`, it should be checked.